### PR TITLE
Add documentation for class_attribute default option

### PR DESCRIPTION
**Pull request by glaucocustodio**
Documentation for option introduced at: https://github.com/rails/rails/pull/29270

### CODE REVIEW

File: guides/source/active_support_core_extensions.md [hunk 1]
```diff
@@ -946,6 +946,18 @@ A.new.x = 1
 A.new.x # NoMethodError
 ```
 
+A default value can be set using the option `:default`.
+
+```ruby
+module ActiveRecord
+  class Base
+    class_attribute :table_name_prefix, default: "my"
```
Comment (kaspth) [inline guides/source/active_support_core_extensions.md]: This seems to subtlety suggest you can set an app-wide table name prefix like this. Let's just replace:

```ruby
module ActiveRecord
   class Base
     class_attribute :table_name_prefix, instance_writer: false, default: ""
   end
 end
```

in the example above. No need to elaborate with any text. `:default` is pretty self-explanatory :)

Comment (glaucocustodio) [inline guides/source/active_support_core_extensions.md]: Done ;)

**Comment by vipulnsward** [comment]
Can you squash the commits?

**Comment by sgrif** [comment]
Can you also update the documentation to list the option, and explain what it does? I think there would be value in having more than just the usage example

**Comment by glaucocustodio** [comment]
Added documentation for list of options, please check.

File: activesupport/lib/active_support/core_ext/class/attribute.rb [hunk 2]
```diff
@@ -7,6 +7,16 @@ class Class
   # Declare a class-level attribute whose value is inheritable by subclasses.
   # Subclasses can change their own value and it will not impact parent class.
   #
+  # ==== Options
+  #
+  # * <tt>:instance_reader</tt> - Sets the instance reader method (defaults to true).
+  # * <tt>:instance_writer</tt> - Sets the instance writer method (defaults to true).
+  # * <tt>:instance_accessor</tt> - Sets both instance methods (defaults to true).
+  # * <tt>:instance_predicate</tt> - Sets a predicate method (defaults to true).
+  # * <tt>:default</tt> - Sets a default value for the attribute (defaults to nil).
```
Comment (kaspth) [inline activesupport/lib/active_support/core_ext/class/attribute.rb]: @sgrif how do these make the usage examples clearer? I don't see how a reader would understand e.g. "Sets the instance writer method."

**Pull request merged**